recipe/meta.yaml

```diff
@@ -63,7 +67,6 @@ outputs:
         - libabseil-static ={{ version }}=cxx{{ cxx_standard }}*
         # make sure we don't co-install with old version of old package name
         - abseil-cpp ={{ version }}
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
```

Comment:
Why was this removed?

Reply:
It was my mistake; I confused with that the run-export would cover it. Fixed in 43e015bc8f7a4ff949cba37d61d138b928808474